src/tools/kata-ctl/src/check.rs

```diff
@@ -89,13 +95,49 @@ pub fn run_network_checks() -> Result<()> {
     Ok(())
 }
 
+fn get_kata_version_by_url(url: &str) -> Result<String, reqwest::Error> {
```

Comment:
Nit: This might be clearer as shown since `anyhow` normally only requires a single param to `Result`:

```suggestion
fn get_kata_version_by_url(url: &str) -> std::result::Result<String, reqwest::Error> {
```

Reply:
@jodh-intel thanks